docs/products/kafka/concepts/acl.md

```diff
@@ -1,116 +1,216 @@
 ---
-title: Access control lists and permission mapping
+title: Access Control Lists in Aiven for Apache Kafka®
+sidebar_label: Access control lists
 ---
 
-Aiven for Apache Kafka® uses **access control lists** (ACL) and user definitions to establish individual rights to produce, consume or manage topics.
-To manage users and ACL entries, you can access the
-corresponding options in the left-side navigation menu on the service
-page within the [Aiven Console](https://console.aiven.io/). For detailed
-instructions, see
-[Manage users and access control lists](/docs/products/kafka/howto/manage-acls).
+Access Control Lists (ACLs) in Aiven for Apache Kafka® manage access to topics, consumer groups, clusters, and Schema Registry with permissions.
 
-## ACL structure
+Aiven supports two ACL models:
 
-The ACL consists of **ACL entries**. An ACL entry is defined as the
-combination of:
+- **Aiven ACLs**: Simplified topic-level access control with basic permissions and
+  wildcard support.
+- **Kafka-native ACLs**: Advanced resource-level access control with fine-grained
+  permissions, including `ALLOW` and `DENY` rules.
 
--   the username
--   the permission given to the user
--   the associated topics
+## Aiven ACL capabilities
 
-The username portion of the ACL entry can be an Apache Kafka® service
-user name, or a string containing wildcards, which can match multiple
-users. Similarly, the topic portion can be a single Apache Kafka® topic
-name or can use a wildcard pattern. The permission is one of `read`,
-`write`, `readwrite` and `admin`.
+Aiven ACLs provide a simpler way to manage access to Apache Kafka topics and Schema
+Registry resources. It is ideal for use cases requiring basic permissions and wildcard
+support.
 
-The wildcards supported are:
+- **Permissions**: Allow `read`, `write`, `readwrite`, or `admin` permissions to be
+  assigned to specific users or groups at the topic level.
+- **Wildcard support**: Simplifies access control by allowing patterns for usernames
+  and resource names, such as `logs-*` for topics or `user?` for usernames.
+- **User-specific or group-level permissions**: Provides precise access control by
+  applying permissions to specific users or groups. For instance, `username: analyst*`
+  applies to all usernames starting with "analyst."
```

Comment:
Same as above, this explains "group-level" permission but as there are no groups as separate entities I think it should be some other term here. The matching example is a good one here.
```suggestion
- **User-specific permissions**: Provides precise access control by
  applying permissions to matching users. For instance, `analyst*`
  applies to all usernames starting with "analyst".
```

But maybe full stop should be outside quotes for technical exactness even if grammatically it could go inside quotes?

My suggestions are just something to consider, not sure how it is really best explained